Cargo.toml

```diff
@@ -18,8 +18,9 @@ scraper = "0.12.0"
 regex = "1.5.4"
 
 [dependencies.songbird]
-version = "0.2.0"
-features = ["builtin-queue"]
+git = "https://github.com/serenity-rs/songbird"
+branch = "current"
```

Comment:
Still, test it with songbird 2.0 maybe, it is not great to be using bleeding edge master version.

Reply:
Tried it, doesn't compile because it has no support for a "yt-dlp" feature. 😐